test/hotspot/gtest/runtime/test_virtualMemoryTracker.cpp

```diff
@@ -432,11 +435,10 @@ class VirtualMemoryTrackerTest {
   }
 
   static void test_remove_uncommitted_region() {
-    RegionsTree* rtree = VirtualMemoryTracker::Instance::tree();
-    rtree->tree().remove_all();
-
     size_t size  = 0x01000000;
     ReservedSpace rs = MemoryReserver::reserve(size, mtTest);
+
+    RegionsTree* rtree = VirtualMemoryTracker::Instance::tree();
```

Comment:
I have to confess to not paying attention to this change, but shouldn't you fetch the tree under the lock also?

Reply:
Okay, I see that too.  The code just made me wonder but it's okay.